src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -1045,6 +1045,54 @@ const schema = {
     },
     primaryPhone: {
       $ref: '#/definitions/phone'
+    },
+    homelessOrAtRisk: {
+      type: 'string',
+      'enum': [
+        'No',
+        'homeless',
+        'atRisk'
+      ]
+    },
+    homelessHousingSituation: {
+      type: 'string',
+      'enum': [
+        'I’m living in a homeless shelter.',
```

Comment:
Do we think we'll end up with codes to send to EVSS for this? I'm a little wary of descriptive strings in `enum` fields like this.

Reply:
I think we will need to send them specific values for this, but it's not clear what that'll be yet.

Reply:
Right...I should probably switch to at least using labels for the full text and tightening up the enums.